addons/project/tests/test_res_config_settings.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import tagged, Form, TransactionCase
+
+
+@tagged('post_install', '-at_install')
+class TestResConfigSettings(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.analytic_plan_1 = cls.env['account.analytic.plan'].create({
+            'name': 'Plan 1',
+            'default_applicability': 'optional',
+            'company_id': False,
+        })
+        cls.analytic_plan_2 = cls.env['account.analytic.plan'].create({
+            'name': 'Plan 2',
+            'default_applicability': 'optional',
+            'company_id': False,
+        })
```

Comment:
we could create in batch. 🙂 
```suggestion
        cls.analytic_plan_1, cls.analytic_plan_2 = cls.env['account.analytic.plan'].create([
            {
                'name': 'Plan 1',
                'default_applicability': 'optional',
                'company_id': False,
            }, {
                'name': 'Plan 2',
                'default_applicability': 'optional',
                'company_id': False,
            },
        ])
```